MongoDB.Entities/Core/FileEntity.cs

```diff
@@ -41,25 +39,19 @@ public abstract class FileEntity : Entity
         /// If this value is set, the uploaded data will be hashed and matched against this value. If the hash is not equal, an exception will be thrown by the UploadAsync() method.
         /// </summary>
         [IgnoreDefault]
-        public string MD5 { get; set; }
-
-
-        /// <summary>
-        /// Access the DataStreamer class for uploading and downloading data
-        /// </summary>
-        public DataStreamer Data => streamer ??= new DataStreamer(this);

```

Comment:
do we need to break the api here? or will you be adding this back in later?

Reply:
yes, since `DataStreamer` needs to get `FileChunk` collection, it needs to know about the `DBContext`, so we would either have to 

1. pass the `DBContext` to the `FileEntity` just to be used in `new DataStreamer(this, db);`
2. or leave the creation of `DataStreamer` to the `DBContext` (better)